src/foam/mlang/mlang.js

```diff
@@ -2696,6 +2701,22 @@ foam.CLASS({
 
         getArray().add(a);
       `
+    },
+    {
+      name: 'toString',
+      code: function() {
+        var projectionString = this.cls_.name + '(' + this.exprs.join(',') + ')';
```

Comment:
Just combine these two lines since you don't use the var.

Reply:
fixed